app.yaml

```diff
@@ -1,3 +1,12 @@
 # dit bestand laat ons toe de app te configureren.
 #definieer welke runtime we willen gebruiken:
-runtime: python37
\ No newline at end of file
+runtime: python37
+
+#definieer type van scaling (F=autoscaling)
+instance_class: F2
+
+#Hoe URL patronen verwerkt moeten worden
+handlers:
+- url: /.*
+  secure: always
+  script: auto
```

Comment:
this way you will still deploy your default service.
try to create a new service